src/structures/BaseCommandInteraction.js

```diff
@@ -123,6 +125,14 @@ class BaseCommandInteraction extends Interaction {
       }
     }
 
+    if (attachments) {
+      result.attachments = new Collection();
+      for (const attachment of Object.values(attachments)) {
```

Comment:
```suggestion
      for (const attachment of attachments.values()) {
```
unless i'm misunderstanding, attachments is a collection